alcove/query/api.py

```
@@ -19,8 +30,144 @@
+            "uptime_seconds": uptime_s,
+        }
+
 app.mount("/static", StaticFiles(directory=str(STATIC_DIR)), name="static")
 templates = Jinja2Templates(directory=str(TEMPLATES_DIR))
+templates.env.globals["alcove_title"] = os.environ.get("ALCOVE_TITLE", "Alcove")
+templates.env.globals["alcove_tagline"] = os.environ.get("ALCOVE_TAGLINE", "Document Search")
+templates.env.globals["alcove_logo_text"] = os.environ.get("ALCOVE_LOGO_TEXT", "A")
+templates.env.globals["alcove_accent_color"] = os.environ.get("ALCOVE_ACCENT_COLOR", "")
+templates.env.globals["alcove_accent_color_light"] = os.environ.get("ALCOVE_ACCENT_COLOR_LIGHT", "")
+
+_default_footer = f"{os.environ.get('ALCOVE_TITLE', 'Alcove')} \u00a9 {datetime.now().year}"
```

Ruff (0.15.5)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)